helm_deploy/values-dev.yaml

```diff
@@ -15,6 +15,7 @@ ingress:
   path: /
 
 env:
+  ENV: dev
```

Comment:
we already have an env var that does this job - `NODE_ENV`. it's set in the dockerfile. can we reuse that for this purpose?

Reply:
unless this is specifically for sentry - in which case can we rename to `SENTRY_ENV` or similar?

Reply:
Good question, I was thinking
- I named this `SENTRY_ENV` in the service code, but semantically, I'm describing what environment is the application is running in (~ `APP_ENV`)
- `NODE_ENV` would be similar to `RAILS_ENV`, `MIX_ENV` etc. typically used in other frameworks, so I'd be happy to use that -- is there a change in behaviour if it's not set to `production` everywhere?

Reply:
specifically, I'm concerned with the semantic difference

we have code like this, which seems to allude to "how" the app is running, locally or during testing
```
process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test'
```

Whereas this new variable is more about what type of 'deployed' environment it is. It's not _specific_ to Sentry, but we do use it here first (I imagine AppInsights would be also interested?)

So I guess, which one would we prefer, `SENTRY_ENV` or `DEPLOYMENT_ENV`?

Reply:
Does 62ad70dcd848256eca5d65e1088e96f7a99867e4 make sense?